common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -539,11 +554,16 @@ export class ActiveRow implements LayoutRow {
   ) {
     // Apply defaults, setting the width and other undefined properties for each key
     let keys=row['key'];
+    const DEFAULT_KEY = ActiveKeyBase.DEFAULT_KEY;
```

Comment:
This is now ActiveKeyBase.DEFAULT_KEY and not ActiveKey.DEFAULT_KEY?

Reply:
`ActiveKey` extends `ActiveKeyBase`, so TS didn't raise a fuss about it - either before or after.  (It is a static member, so it's kinda inherited.)  It's probably best to reference the actual class supporting its definition, though.